packages/principles-core/src/runtime-v2/runner/__tests__/m6-06-real-path.test.ts

```
@@ -0,0 +1,542 @@
+      }
+
+      const result = await runPdCli(
+        [
+          'runtime',
+          'probe',
+          '--runtime',
+          'openclaw-cli',
+          '--openclaw-local',
+          '--json',
+        ],
+        testWorkspace,
```

ESLint

[error] 'testWorkspace' is not defined.

(no-undef)